warzone/Player.cpp

```diff
@@ -1 +1,110 @@
 #include "Player.h"
+
+Player::Player() {  //default constructor
+	this->playerTerritories = new vector<Territory*>();
+	this->playerHand = new Cards::Hand();
+	this->playerOrdersList = new OrdersList();
+}
+
+
+Player::Player(vector<Territory*>* territoriesToAdd, OrdersList* playerList, Cards::Deck* deckToTakeFrom) { //Constructor with all parameters
+	this->playerTerritories = territoriesToAdd;
+	this->playerOrdersList = playerList;
+	this->playerHand = new Cards::Hand(deckToTakeFrom);
+}
+
+Player::~Player() {  //destructor 
+	for (Territory* t : *playerTerritories) {
+		delete t;
+	}
```

Comment:
You don't have to iterate through and delete each territory, they will be deleted when Map is deleted. But you can still delete playerTerritories

Reply:
sounds good!